go/obscuronode/host/p2p/p2p.go

```diff
@@ -181,6 +175,6 @@ func (p *p2pImpl) sendBytes(address string, tx []byte) {
 
 	_, err = conn.Write(tx)
 	if err != nil {
-		panic(err)
+		nodecommon.LogWithID(p.nodeID, "could not send message to peer on address %s: %v", address, err)
```

Comment:
do we want to do this?

Reply:
I think so. It fits with general behaviour here, which is not to freak if we get a bad connection.